client/verta/verta/_deployment/endpoint.py

```diff
@@ -29,10 +31,15 @@ def _create(cls, conn, conf, workspace, path, description=None):
 
     @classmethod
     def _create_json(cls, conn, workspace, path, description=None):
+        json = {}
+        if description:
+            json["description"] = description
         if not path.startswith('/'):
             path = '/' + path
-
-        raise NotImplementedError
+        json["path"] = path
+        response = _utils.make_request("POST", "{}://{}{}".format(conn.scheme, conn.socket, "/api/v1/deployment/workspace/{}/endpoints".format(workspace)), conn, json=json)
+        assert response.status_code == 200
```

Comment:
I failed to mention this in the ticket! We have a util for checking successful HTTP responses.
```suggestion
        _utils.raise_for_http_error(response)
```